libs/ledger-live-common/package.json

```diff
@@ -131,6 +131,7 @@
     "@elrondnetwork/erdjs": "11.0.0",
     "@elrondnetwork/erdjs-network-providers": "^1.1.2",
     "@hashgraph/sdk": "2.14.2",
+    "@ipld/dag-cbor": "^9.2.0",
```

Comment:
I don't find it is used in the current code. Isn't it?

Reply:
It's definitely not, now removed from package.json... it was an accident 😅